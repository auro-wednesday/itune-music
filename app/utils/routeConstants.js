export default {
  repos: {
    route: '/',
    props: {
      maxwidth: 500,
      padding: 20
    },
    exact: true
  },
  Itunes: {
    route: '/itunes',
    props: {
      maxwidth: 500,
      padding: 20
    },
    routes: [
      {
        route: '/itunes/trackName',
        props: 'itunesData'
      }
    ],
    exact: true
  }
};
