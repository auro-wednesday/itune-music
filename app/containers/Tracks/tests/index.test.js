/**
 *
 * Tests for Tracks
 *
 *
 */

// import React from 'react';

// import { fireEvent } from '@testing-library/dom';
import { TracksTest as Tracks } from '../index';

import { renderWithIntl } from '@app/utils/testUtils';

describe('<Tracks/>', () => {
  it('should render tracks component', () => {
    const { baseElement } = renderWithIntl(() => Tracks());

    expect(baseElement).toMatchSnapshot();
  });
});
