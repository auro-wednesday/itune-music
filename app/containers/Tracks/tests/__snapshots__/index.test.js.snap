// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tracks/> should render tracks component 1`] = `
<body>
  <div />
</body>
`;
